doc/nrf/libraries/bluetooth_services/mesh/models.rst

```diff
@@ -3,15 +3,17 @@
 Bluetooth Mesh models
 #####################
 
-Nordic Semiconductor provides a variety of model implementations from the `Bluetooth Mesh model specification`_ and their API documentation, as well as vendor-specific models.
+Nordic Semiconductor provides a variety of Bluetooth® Mesh model implementations from the `Bluetooth Mesh model specification`_ and their API documentation, as well as vendor-specific models.
+For more information about these and other models, see also :ref:`ug_bt_mesh_overview_models` and `Bluetooth Mesh model overview`_.
 
-For more information about these and other models, see also `Bluetooth Mesh model overview`_.
+In addition to model implementations in the |NCS|, there are models implemented in Zephyr (see :ref:`zephyr:bluetooth_mesh_models`).
```

Comment:
I think we should clarify here that:
Models in Bluetooth Mesh Protocol + DFU&MBT => Zephyr
Models in Bluetooth Mesh models => NCS